docs/examples/resizableCarousel.md

```diff
@@ -1,8 +1,11 @@
-## Container wrapper
+## Resizable carousel 
 You can surround the Carousel by a container, which can be resized
 independently of the window. You should only make sure,
 that the container doesn't exceed the viewport.
 ```jsx render
+// import Carousel from '@brainhubeu/react-carousel';
```

Comment:
why is it commented out?

Reply:
and in other places

Reply:
In gatsby-docs-kit imports are managed in globalReferences file, therefore, we didn't need them here, but they are required in day to day usage

Reply:
ok